src/scss/components/multi-menu/_multi-menu.scss

```
@@ -257,3 +257,33 @@
     }
   }
 }
+
+// Search box styling for devices and mobile views.
+// Hide the extra search box everywhere
+// -----------------------------------------------------------------------
+// Hide the search box before moved - in case more than one is added.
+nav {
+  &+ .su-site-search {
+    display: none;
+  }
+}
+
+// Hide the moved box
+ul {
+  li {
+    .su-site-search {
+      display: none;
+    }
+  }
+}
+
+.su-multi-menu {
+  // show the extra search box in responsive menu.
+  @include grid-media-between('xs', 'md') {
+    div[data-drupal-selector='search-block-form-2'] {
```

```suggestion
    .su-site-search {
```

search-block-form-2 doesn't exist for me.

---

Oh man. I'm so sorry. :( I'll fix.